source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py

```diff
@@ -389,30 +388,34 @@ def __init__(
             fields = [self.address_field, self.user_field, self.password_field]
         else:
             fields = [self.address_field]
-        
+
+        btn_login = widgets.Button(
+            description="Login",
+            disabled=False
+        )
+
+        def login_to_iqs(_):
+            try:
+                self.iqs = self.connect()
+                server_status = self.iqs.server_management.get_server_status_with_http_info()
+
+                if server_status[1] == 200:
+                    if server_status[0].component_statuses["SERVER"] == "UP":
+                        print("Connected! IQS is ready to use.")
```

Comment:
one more thing: will this be printed multiple times, if I click the button repeatedly (with or without changing credentials)?

perhaps instead of printing it, there should be a Label widget in a HBox with the button, the the connection message could be set as the content of the label widget.